src/ripple/app/tests/TxQ_test.cpp

```diff
@@ -122,7 +122,7 @@ class TxQ_test : public TestSuite
     }
 
 public:
-    void run()
+    void testQueue()
```

Comment:
Interesting. gcc 5.2 took the `return p;`. Clang 3.7 choked on it. So clang at least needs the `std::move`.